cartoframes/context.py

```diff
@@ -167,6 +166,9 @@ def read(self, table_name, limit=None, index='cartodb_id',
 
         return self.query(query, decode_geom=decode_geom)
 
+    def datasets(self):
+        return DatasetManager(self.auth_client).all()
```

Comment:
Nice :) Since this returns a list of `Dataset` instances, we should probably have it be a list of names instead.

```python
return [d.name for d in DatasetManager(...).all()]
```

Another thing is that `DatasetManager().all()` is slow for my account (~1250 tables). Is there another way to list tables that's faster? My guess is that we should avoid querying the information schema table, but this would probably be faster.

Reply:
Ok. Indirectly, you've just answered my biggest doubt: I wasn't sure if a CARTO Frames function (`datasets`) should return instances of a class that is defined in CARTO Python SDK. I'll take a look at it because only the name is going to be returned, we should reduce the retrieved information a lot (therefore making it faster).

Reply:
Yeah, good point about carto python sdk `Dataset` object. One thing that we can think about for the near future planning is how this can look for the cartoframes Analysis API's data source objects: http://cartoframes.readthedocs.io/en/prototype-table-objects/cartoframes.analysis.html#data-sources

Should the returned datasets be a to-be-created `Table` instance? Or should we hold off for now, and just have it as a string for the table name, and then update it to `Table` later once we write the spec? I support this second option.